processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -16,7 +16,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.naming;
+package org.mapstruct.ap.spi;
```

Comment:
Hehe... This is what I suggested back then when we introduced the interface, but we decided to not expose the default implementation - I don't fully remember the reasons. But I'm fine with it as it is now.